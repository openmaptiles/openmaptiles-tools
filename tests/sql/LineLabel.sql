-- LineLabel
SELECT LineLabel(14, 'Foobar', ST_GeomFromText('POINT(0 0)',900913)) AS t_v1;	-- t
SELECT LineLabel(14, 'Foobar', ST_GeomFromText('LINESTRING(0 0, 0 300)',900913)) AS f_v2;	-- f
SELECT LineLabel(15, 'Foobar', ST_GeomFromText('LINESTRING(0 0, 0 300)',900913)) AS t_v3;	-- t
