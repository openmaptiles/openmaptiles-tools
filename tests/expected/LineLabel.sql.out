 t_v1 
------
 t
(1 row)

 f_v2 
------
 f
(1 row)

 t_v3 
------
 t
(1 row)

